From e7973e352b8649b10053f3feb82134d9cefb851e Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Tue, 17 May 2022 19:00:02 +0200
Subject: [PATCH] meson.build: make epoll-shim dependency optional

After updating my host system, meson.build tries to depend on the epoll-shim.
Since we are not on BSD, we dont have it and it is not needed. Make the
dependency on it optional.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 84abdcb..9687b23 100644
--- a/meson.build
+++ b/meson.build
@@ -35,7 +35,7 @@ wayland_cursor = dependency('wayland-cursor')
 epoll = dependency('', required: false)
 if (not cc.has_function('timerfd_create', prefix: '#include <sys/timerfd.h>') or
     not cc.has_function('signalfd', prefix: '#include <sys/signalfd.h>'))
-	epoll = dependency('epoll-shim')
+	epoll = dependency('epoll-shim', required: false)
 endif
 
 if get_option('sd-bus-provider') == 'auto'
-- 
2.25.1

